src/templates/pages/home.pug

```diff
@@ -91,20 +91,14 @@
                 h3.text-secondary.mb-2 Smart Tracing That Keeps Communities Safe
                 p.lead.mb-0.text-white.opacity-8 Built for real humans in need by real humans who care.
               .col-lg-5.text-lg-right.mt-4.mt-lg-0
-                a.btn.btn-white.btn-icon.my-2(href='https://www.github.com/zerobase-io/smart-tracing/wiki', target='_blank')
```

Comment:
Should documentation point to the wiki?

Reply:
I think the idea is that we get rid of documentation - which I believe is already merged into master (I got rid of it in the footer update branch)